client/src/components/DetailView.tsx

```diff
@@ -44,6 +47,28 @@ const SocialShareDetailView = () => (
   />
 );
 
+const LearnMoreAccordion = () => (
+  <I18n>
+    {({ i18n }) => (
+      <Accordion title={i18n._(t`Learn more`)} titleOnOpen={i18n._(t`Close`)}>
+        <br />
+        <Trans>
+          <p>
+            While the legal owner of a building is often an “LLC” company, these names and business
+            addresses registered with HPD offer a clearer picture of who the landlord really is.
+          </p>
+          <p>
+            People listed here as “Head Officer” or “Owner” usually have ties to building ownership,
+            while “Site Managers” are part of management. That being said, these names are self
+            reported, so they can be misleading.
+          </p>
+          <p>Learn more about HPD registrations and how they power this tool on the About page.</p>
```

Comment:
Should "About page" be hyperlinked to the about page?

Reply:
yes!!